applications/ShapeOptimizationApplication/python_scripts/mapper_factory.py

```diff
@@ -39,9 +39,36 @@ def CreateMapper(origin_model_part, destination_model_part, mapper_settings):
         "revolution_settings"        : {
             "point" : [0.0, 0.0, 0.0],
             "normal": [0.0, 0.0, 0.0]
+        },
+        "adaptive_filter_settings"   : {
```

Comment:
What do you think about adding "adaptive_filter_method"?? see Shape Module.

Then, we can have different methods later, how we compute the radius: based on curvature or element size or smth else in future.